canonical-web-and-design/hiring.canonical.com
# Make performance chart more generic

## Done
- Performance chart moved to shared folder.
- Performance chart is updated so that it can be added other stages easily.
## QA
Setup:
- Start the application: `dotrun`
Validation:
- Verify performance chart is displayed correctly from `http://0.0.0.0:8501/stages/applications-review`
## Issue
Fixes #149
